packages/flame/lib/src/effects/sequence_effect.dart

```diff
@@ -136,7 +197,7 @@ class _SequenceEffectEffectController extends EffectController {
   }
 
   @override
-  double get progress => (_index + 1) / n;
+  double get progress => (_index < 0 ? -_index : _index + 1) / n;
```

Comment:
Can you explain this a bit? In what cases is the `_index` negative? This doesn't seem like it would follow how `progress` works for the rest of the effects.

Reply:
The way the `SequenceEffect` has been written for some time now is that it uses a negative `_index` to determine if the pattern is in its reverse phase.